src/config/kernel.md

```diff
@@ -22,44 +17,25 @@ modules.
 ## Removing old kernels
 
 When updating the kernel, old versions are left behind in case it is necessary
-to roll back to an older version. Over time, many old kernel version accumulate
-and make updating dmks modules take a long time. Thus, it may be advisable to
-clean old kernels from time to time.
+to roll back to an older version. Over time, many old kernel versions can
+accumulate, taking up a lot of space and making dkms modules updates take a long
+time. Thus, it may be advisable to clean old kernels from time to time.
```

Comment:
Suggest: "versions can accumulate, consuming disk space and increasing the time needed to do dkms module updates. Thus, it may be advisable to remove ..."

Reply:
I changed it to "and increasing the time taken by dmks module updates"